src/headers.zig

```diff
@@ -38,20 +33,17 @@ pub const Headers = struct {
     }
 
     pub fn appendValue(self: *Headers, name: []const u8, value: []const u8) !void {
```

Comment:
This function now needs documentation that it will not duplicate the memory it is passed, and that ownership of the memory remains the caller's responsibility.